src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -9147,7 +9142,7 @@ void VRFTerminalUnitEquipment::ControlVRFToLoad(EnergyPlusData &state,
         // If the QZnReq <= FullOutput the unit needs to run full out
         if (QZnReq <= FullOutput) {
             // if no coil present in terminal unit, no need to reset PLR?
-            if (thisVRFTU.CoolingCoilPresent) {
+            if (thisVRFTU.CoolingCoilPresent && DXCoolingCoilOprCtrl) {
                 PartLoadRatio = 1.0;
```

Comment:
The other simple change at line 9145 was to only set the cooling coil PLR to 1 when QZnReq <= FullOutput was when the cooling coil was allowed to operate.